### DISCUSSION TITLE
tracking demo

### DISCUSSION THREAD

**Discussion by andreaceruti** [facebookresearch/detectron2#4092] [General]
I have seen that you have included a tracking folder, how do we use it while doing inference on video frames??

**Comment by zhanghang1989** [comment]
@wenliangzhao2018

**Comment by wenliangzhao2018** [comment]
Can you refer examples in the unit tests: https://github.com/facebookresearch/detectron2/blob/main/tests/tracking/test_bbox_iou_tracker.py

**Reply by andreaceruti** [reply]
Ok thank you @wenliangzhao2018!! I will try to figure it out how to set up all.
Anyway I think that a more verbose tutorial could be added to the tutorial notebook of Detectron2, for example my application consists in training a MaskRCNN model, then do inference and tracking on separated video frames (so with no annotations). I will share it to you a notebook as example If i will be able to add the tracking

**Comment by StudioTV** [comment]
Hey ! 

Thank you for this feature, is it possible to have a clear example on how to use it ? I tried to figure it out with the unit tests, but I can't apply it on my own video.

Thank you guys,
